docs/sources/writing-guide/image-guidelines/index.md

```diff
@@ -46,11 +46,10 @@ Use the following recommendations when you size images:
 
 -  **Scope**: Limit the contents of an image to the relevant portion.
 Do not include distracting or unnecessary content and whitespace.
--  **Format**: Because they are lossless, **PNG** and **SVG** are the preferred image formats.
-We do not recommend JPG images because they are lossy and might look blurry.
+-  **Format**: **png** and **svg** are the preferred image formats. Use **jpg** only for photos.
```

Comment:
```suggestion
-  **Format**: **PNG** and **SVG** are the preferred image formats. Use **JPG** only for photos.
```

Reply:
I've intentionally changed these to lowercase as we use case-sensitive filesystems and want to encourage consistency with the web team's style.